src/osemgrep/cli_scan/Scan_subcommand.ml

```diff
@@ -199,6 +199,11 @@ let run (conf : Scan_CLI.conf) : Exit_code.t =
       let rules, errors =
         Rule_fetching.partition_rules_and_errors rules_and_origins
       in
+      let rules =
```

Comment:
Maybe add a comment like (* TODO: we should probably warn the user about rules using the same id *)

Reply:
done in [e5962c4](https://github.com/returntocorp/semgrep/pull/7774/commits/e5962c4a46ef2195ca60a2fdbebfb7461875323b)